test/e2e/network_test.go

```diff
@@ -148,6 +153,31 @@ func testEastWestNetworking(t *testing.T) {
 	}
 }
 
+// getDeploymentLogs ensures logs of multiple deployments
+// are collected via a pod labelSelector
+func (tc *testContext) getDeploymentLogs(deployments ...*appsv1.Deployment) {
```

Comment:
No sure about the `get` prefix in the function name `getDeploymentLogs`, since void return. As per inline documentation it `"ensures logs ..."`